engine/crates/engine-v2/schema/src/lib.rs

```diff
@@ -135,6 +135,11 @@ impl Schema {
             .map(|pos| FieldDefinitionId::from(usize::from(fields.start) + pos))
     }
 
+    pub fn iter_graphql_endpoints(&self) -> impl ExactSizeIterator<Item = sources::graphql::GraphqlEndpointWalker<'_>> {
```

Comment:
The method would have been better placed in the `SchemaWalker`:
https://github.com/grafbase/grafbase/blob/main/engine/crates/engine-v2/schema/src/walkers/mod.rs#L119